src/MOI_wrapper.jl

```diff
@@ -431,5 +433,120 @@ end
+    return
+end
+
+function _ensure_conflict_computed(model::Optimizer)
+    if model.conflict === nothing
+        error("Cannot access conflict status. Call `CPLEX.compute_conflict(model)` first. " * 
+              "In case the model is modified, the computed conflict will not be purged.")
+    end
+end
+
+"""
+    ConflictStatus()
+
+Return an `MOI.TerminationStatusCode` indicating the status of the last computed conflict.
+
+If a minimal conflict is found, it will return `MOI.OPTIMAL`. If the problem is feasible, it will
+return `MOI.INFEASIBLE`. If `compute_conflict` has not been called yet, it will return 
+`MOI.OPTIMIZE_NOT_CALLED`. 
+"""
+struct ConflictStatus <: MOI.AbstractModelAttribute  end
+MOI.is_set_by_optimize(::ConflictStatus) = true
+
+function MOI.get(model::Optimizer, ::ConflictStatus) 
+    if model.conflict === nothing
+        return MOI.OPTIMIZE_NOT_CALLED
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_MINIMAL
+        return MOI.OPTIMAL
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_FEASIBLE
+        return MOI.INFEASIBLE
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_CONTRADICTION
+        return MOI.OTHER_LIMIT
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_DETTIME_LIM
+        return MOI.TIME_LIMIT
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_IT_LIM
+        return MOI.ITERATION_LIMIT
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_MEM_LIM
+        return MOI.MEMORY_LIMIT
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_NODE_LIM
+        return MOI.NODE_LIMIT
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_OBJ_LIM
+        return MOI.OBJECTIVE_LIMIT
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_TIME_LIM
+        return MOI.TIME_LIMIT
+    elseif model.conflict.stat == CPX_STAT_CONFLICT_ABORT_USER
+        return MOI.OTHER_LIMIT
+    else
+        return MOI.OTHER_LIMIT
+    end
+end
+
+function MOI.supports(::Optimizer, ::ConflictStatus)
+    return true
+end
+
+"""
+    ConstraintConflictStatus()
+
+A Boolean constraint attribute indicating whether the constraint participates in the last computed conflict.
+"""
+struct ConstraintConflictStatus <: MOI.AbstractConstraintAttribute end
+MOI.is_set_by_optimize(::ConstraintConflictStatus) = true
+
+function _sinvar_get_conflict_status(model::Optimizer, index::MOI.ConstraintIndex)
+    _ensure_conflict_computed(model)
+    var_in_conflict = findfirst(isequal(LQOI.get_column(model, model[index]) - 1), model.conflict.colind)
+
+    if var_in_conflict === nothing
+        return nothing
+    else
+        return model.conflict.colstat[var_in_conflict]
+    end
+end
 
+function MOI.get(model::Optimizer, ::ConstraintConflictStatus, index::MOI.ConstraintIndex{<:MOI.SingleVariable, <:LQOI.LE}) 
+    status = _sinvar_get_conflict_status(model, index)
+    return status !== nothing && (status == CPLEX.CPX_CONFLICT_MEMBER || status == CPLEX.CPX_CONFLICT_UB)
```

Comment:
So what to do about the `POSSIBLE` statuses?

Reply:
They are only generated when `ConflictStatus` is not `MOI.OPTIMAL`. How should we behave in this case? One sensible option would be to discard the distinction between the _POSSIBLE_ and the standard statuses (after all, the user is warned that the result is not optimal). The other option I see is a trilean: true/maybe/false.

Reply:
Maybe we should just add a note to `generate_conflict` and add a TODO about the possible statuses.